.gitmodules

```diff
@@ -6,7 +6,45 @@
 	path = scala2-library
 	url = https://github.com/lampepfl/scala.git
 	branch = dotty-library2.12
-[submodule "collection-strawman"]
-	path = collection-strawman
-	url = https://github.com/dotty-staging/collection-strawman.git
-	branch = dotty
+[submodule "community-build/community-projects/algebra"]
+	path = community-build/community-projects/algebra
+	url = https://github.com/dotty-staging/algebra
+[submodule "community-build/community-projects/betterfiles"]
+	path = community-build/community-projects/betterfiles
+	url = https://github.com/dotty-staging/better-files
+[submodule "community-build/community-projects/scalacheck"]
+	path = community-build/community-projects/scalacheck
+	url = https://github.com/dotty-staging/scalacheck
+[submodule "community-build/community-projects/ScalaPB"]
+	path = community-build/community-projects/ScalaPB
+	url = https://github.com/dotty-staging/ScalaPB
+[submodule "community-build/community-projects/scalatest"]
+	path = community-build/community-projects/scalatest
+	url = https://github.com/dotty-staging/scalatest
+[submodule "community-build/community-projects/scopt"]
+	path = community-build/community-projects/scopt
+	url = https://github.com/dotty-staging/scopt
+[submodule "community-build/community-projects/squants"]
+	path = community-build/community-projects/squants
+	url = https://github.com/dotty-staging/squants
+[submodule "community-build/community-projects/collection-strawman"]
```

Comment:
I would either copy-paste the sources of the strawman in our test folder or get rid of it entirely, there's no point in keeping it in a submodule since it's a frozen project that won't evolve anymore.